docs/source/model_doc/t5.rst

```diff
@@ -87,6 +87,10 @@ token. T5 can be trained / fine-tuned both in a supervised and unsupervised fash
 
 .. code-block::
 
+    from transformers import T5ForConditionalGeneration, T5Tokenizer
+    mname = "t5-small"
+    model = T5ForConditionalGeneration.from_pretrained(mname)
+    tokenizer = T5Tokenizer.from_pretrained(mname)
```

Comment:
```suggestion
    from transformers import T5ForConditionalGeneration, T5Tokenizer
    model = T5ForConditionalGeneration.from_pretrained("t5-small")
    tokenizer = T5Tokenizer.from_pretrained(t5-small")
```

Reply:
Could we also add this in the "Unsupervised denoising training" example?

Reply:
of course, will validate the code and add it.